design-library/src/components/BccAlert/BccAlert.vue

```diff
@@ -41,13 +43,21 @@ const contexts = {
     class: "bcc-alert-danger",
     icon: ErrorFillIcon,
   },
+  notice: {
```

Comment:
In the Badge/Pin we've named it `neutral`. I see here it uses the brand colors, so could `brand` be an appropriate option?

I'm mostly thinking about this because the other options are "contexts", which is a specific term in the design system related to the (mostly) semantic colors. Perhaps check this naming with the designers.

Reply:
Fixed! Renamed to neutral